source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -45,6 +46,9 @@ def _get_descriptionFrom(self) -> controlTypes.DescriptionFrom:
 
 	def _get_detailsSummary(self) -> typing.Optional[str]:
 		if not self.hasDetails:
+			# optimisation that avoids having to fetch details relations which may be a more costly procedure.
+			if config.conf["debugLog"]["annotations"]:

```

Comment:
We do, it's more helpful when the debug category is coupled to a module (so that the definition / implementation of `_isDebug` can be shared). In this case it's not so obvious where this would go. Perhaps this will become more obvious when more of the web-annotations system is implemented. For now I think it is better to leave it as it is.